anvil/src/config.rs

```diff
@@ -818,131 +818,141 @@ impl NodeConfig {
                     )
                 };
 
-            let block = provider
-                .get_block(BlockNumber::Number(fork_block_number.into()))
-                .await
-                .expect("Failed to get fork block");
+                let block = provider
+                    .get_block(BlockNumber::Number(fork_block_number.into()))
+                    .await
+                    .expect("Failed to get fork block");
 
-            let block = if let Some(block) = block {
-                block
-            } else {
-                if let Ok(latest_block) = provider.get_block_number().await {
-                    panic!(
-                            "Failed to get block for block number: {fork_block_number}\nlatest block number: {latest_block}"
+                let block = if let Some(block) = block {
+                    block
+                } else {
+                    if let Ok(latest_block) = provider.get_block_number().await {
+                        let mut message = format!(
+                            "Failed to get block for block number: {fork_block_number}\n\
+                        latest block number: {latest_block}"
```

Comment:
We need to remove the whitespace here as it'll show up on the error
```suggestion
latest block number: {latest_block}"
```